specification/schemas/responses/errors/message_batches/DuplicateBatchRequest.yaml

```
@@ -0,0 +1,37 @@
+type: object
+title: Duplicate Request
+additionalProperties: false
+properties:
+  errors:
+    type: array
+    minItems: 1
+    maxItems: 1
+    uniqueItems: true
+    items:
+      type: object
+      additionalProperties: false
+      properties:
+        id:
+          $ref: ../../../types/ErrorIdentifier.yaml
+        code:
+          $ref: ../../../enums/ErrorDuplicateRequest.yaml
+        links:
+          $ref: ../../../types/LinksError.yaml
+        status:
+          type: string
+          enum:
+            - '422'
+          example: '422'
+        title:
+          type: string
+          example: Duplicate batch request
+        detail:
+          type: string
+          example: Request exists with identical messageBatchReference
+        source:
+          type: object
+          additionalProperties: false
+          properties:
+            pointer:
+              type: string
```

Possibly enum so it's seen as a strict value
```suggestion
              type: enum
```

---

I think the pointer has to be a `string` doesn't it, because it references the part of the request body that caused the error, so could be different for each request? It's a string on other error responses

---

That's true, but I was assuming that the intention was that the schema for this response includes the same `pointer` object as all of the other responses? e.g. it has one static value in this response, but can take other values in the other responses? I don't really mind either way tbh

---

Happy to change this to an enum if that's what we want. I chose string to be consistent with the other ones. I agree that there's only one value possible now for this type of error but there are a lot more values possible related to other errors.